### ISSUE TITLE
Is it possible to remove arrow icon?

### ISSUE THREAD

**Issue by stevanpopo** [airbnb/react-dates#2038]
For DateRangePicker, I have found the prop customArrowIcon to have a custom arrow.

Is there a prop to remove the icon completely, such that the div with class .DateRangePickerInput_arrow is not created?

**Comment by mmarkelov** [comment]
@stevanpopo no, there is no prop to do it. It renders here, so there is no control to display this div:
https://github.com/airbnb/react-dates/blob/2b8089bc6ae565eea87d69880cee9b9097b04434/src/components/DateRangePickerInput.jsx#L258-L266
